models/inception_resnet_v1.py

```diff
@@ -300,6 +302,13 @@ def forward(self, x):
         else:
             x = F.normalize(x, p=2, dim=1)
         return x
+    
+    def load_labels(self, pretrained):
+        if pretrained is not 'vggface2':
+            return None
+        with open(os.path.join(os.path.dirname(__file__), f'../data/labels-vggface2.csv')) as f:
```

Comment:
`f-string` is supported only from `python>=3.6`, `Travis CI` also runs on python 3.5
So if you want to use `f-string` you need to drop `python 3.5`  from `CI`, and also updateing `setup.py`.